openpype/hosts/maya/api/plugin.py

```diff
@@ -226,7 +226,17 @@ def update(self, container, representation):
             if alembic_nodes:
                 for attr in alembic_attrs:
                     node_attr = "{}.{}".format(alembic_nodes[0], attr)
-                    alembic_data[attr] = cmds.getAttr(node_attr)
+                    connections = cmds.listConnections(node_attr, plugs=True)
```

Comment:
This should have argument `destinations=False` since otherwise it could also list outputs of the attribute.